tests/char_stamina_test.cpp

```diff
@@ -91,7 +91,7 @@ static int actual_burn_rate( Character &dummy, const move_mode_id &move_mode )
 static void burden_player( Character &dummy, float burden_proportion )
 {
     units::mass capacity = dummy.weight_capacity();
-    int units = static_cast<int>( capacity * burden_proportion / 1_gram );
+    int units = static_cast<int>( std::round( capacity * burden_proportion / 1_gram ) );
```

Comment:
I'm very concerned that this leaves the door open for other errors to persist in the codebase that aren't calling std::round(), this behavior needs to be the same in both builds (though if this is an incremental step to getting things unblocked disregard).

Reply:
Even with fpreset this test is still failing, so I'm adding rounding at the moment.